stdout of test 'tst300` in directory 'src/mal` itself:


# 08:01:13 >  
# 08:01:13 >  Mtimeout -timeout 60 Mserver "--config=/ufs/mk/monet5/Linux/etc/MonetDB5.conf" --debug=10 --set "monet_mod_path=/ufs/mk/monet5/Linux/lib/MonetDB5:/ufs/mk/opensource/MonetDB/Linux/lib/MonetDB" --set "gdk_dbfarm=/ufs/mk/monet5/Linux/var/MonetDB5/dbfarm" --set "sql_logdir=/ufs/mk/monet5/Linux/var/MonetDB5/log" --set mapi_port=42158 --set sql_port=55708 --set monet_prompt= --trace "--config=/ufs/mk/monet5/src/mal/Tests/All.conf" --dbname=mTests_src_mal  tst300.mal </dev/null
# 08:01:13 >  

# Monet Database Server V4.99.19
# Copyright (c) 1993-2004, CWI. All rights reserved.
# Compiled for i686-redhat-linux-gnu/32bit; dynamically linked.
# config:/ufs/mk/monet5/Linux/etc/MonetDB5.conf
# dbfarm:/ufs/mk/monet5/Linux/var/MonetDB5/dbfarm
# dbname:mTests_src_mal
# Visit http://monetdb.cwi.nl/ for further information.
## Exception identifiers can be introduced at any place
#
#	raise myNewException:= "new exception";
#	printf("should not come here\n");
##exception block;
#catch	myNewException;
#	printf("catched exception\n");
#exit	myNewException;
#
function user.main():int;	# 0  (main:int)
# Exception identifiers can be introduced at any place 	# 1  (_2:str)
    raise myNewException := "new exception";	# 2  (myNewException:str)<-(_4:str)
    io.printf("should not come here\n");	# 3 IOprint_str (_5:int)<-(_6:str)
#exception block; 	# 4  (_8:str)
catch myNewException;	# 5  (myNewException:str) jump 7
    io.printf("catched exception\n");	# 6 IOprint_str (_9:int)<-(_10:str)
exit myNewException;	# 7  (myNewException:str)
end main;	# 8  
catched exception

# 08:01:13 >  
# 08:01:13 >  Done.
# 08:01:13 >  

